x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/serializer.ts

```diff
@@ -4,40 +4,133 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
```

Comment:
Question: is this file going to replace the "policy_serialization.ts" in services?

Reply:
Yes, the plan is for policy_serialization to be replaced entirely